### PR TITLE
mapped Order entity

### PR DESCRIPTION
**Pull request by SkaterWeather**

### CODE REVIEW

File: src/main/java/internetshop/dao/hibernate/OrderDaoHibernateImpl.java [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        }
+    }
+
+    @Override
+    public List<Order> getAllOrdersByUserId(Long userId) {
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query query = session.createQuery("FROM User WHERE id=:userId");
+            query.setParameter("userId", userId);
+            User user = (User) query.getSingleResult();
+            query = session.createQuery("FROM Order WHERE user=:user");
```
Comment (Sofasmile) [inline src/main/java/internetshop/dao/hibernate/OrderDaoHibernateImpl.java]: Can we use only one query here? 
FROM Order WHERE user.id=:userId

File: src/main/java/internetshop/model/Order.java [hunk 2]
```diff
@@ -2,11 +2,40 @@
...
 
+@Entity
+@Table(name = "orders")
 public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(columnDefinition = "serial")
     private Long id;
+    @Transient
     private Long userId;
```
Comment (Sofasmile) [inline src/main/java/internetshop/model/Order.java:28]: do we need this field here?

File: src/main/java/internetshop/model/Order.java [hunk 3]
```diff
@@ -30,6 +62,7 @@ public void setId(Long id) {
         this.id = id;
     }
 
+    @Deprecated
```
Comment (Sofasmile) [inline src/main/java/internetshop/model/Order.java:65]: the same)

File: src/main/java/internetshop/dao/hibernate/OrderDaoHibernateImpl.java (hunk 1 above)
Comment (SkaterWeather) [reply src/main/java/internetshop/dao/hibernate/OrderDaoHibernateImpl.java]: yeah, thank

File: src/main/java/internetshop/model/Order.java (hunk 2 above)
Comment (SkaterWeather) [reply src/main/java/internetshop/model/Order.java:28]: yes, there some methods in JDBC impl are using field, if i will remove field, i have to remake OrderDaoJdbcImpl, so it's just better to @Trancient

File: src/main/java/internetshop/model/Order.java (hunk 3 above)
Comment (SkaterWeather) [reply src/main/java/internetshop/model/Order.java:65]: also the same )

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**